src/atoms/SvgIcon/index.js

```diff
@@ -60,6 +61,8 @@ const SvgIcon = props => (
 					return <DoubleArrow {...props} />;
 				case 'star':
 					return <Star {...props} />;
+				case 'avatar':
```

Comment:
Maybe smthing like `DefaultAvatar` or `PlaceholderAvatar` ? I'm not good at naming things
@misund requesting your help :}

Reply:
I think I'd go with `silhouette` or `user`

Reply:
sure, let it be DefaultAvatar

Reply:
In general, I think I would gravitate towards naming icons for what they actually depict (like silhouette, gear, envelope) in the shiny atom, and then adding the metaphor (like user, settings, email) wherever we're using it.

Reply:
So in wolverine-frontend, for example, we could have a component
`const UserIcon = props => <Icon name="silhouette" title="User" />`